app/web/src/store/components.store.ts

```diff
@@ -170,6 +170,7 @@ export type ConnectionMenuData = {
 
 export type ComponentsAndEdges = {
   components: RawComponent[];
+  // TODO REMOVE this is empty now
```

Comment:
This will go away in a followup that removes the serialized type (and JS's use of it). For now, this is always an empty array; in the future, it won't be sent at all.